.eslintrc.js

```diff
@@ -0,0 +1,22 @@
+module.exports = {
+  'env': {
+    'browser': true,
+    'es6': true
+  },
+  'extends': ['eslint:recommended', 'google'],
+  'globals': {
+    'Atomics': 'readonly',
+    'SharedArrayBuffer': 'readonly'
+  },
+  'parserOptions': {
+    'ecmaVersion': 2018
+  },
+  'rules': {
+    'indent': ['error', 2],
+    'linebreak-style': ['error', 'unix'],
+    'quotes': ['error', 'single'],
```

Comment:
I see now what you did here. Ignore my previous comment about quotes in the other pull.

Reply:
:+1: